packages/perseus/src/widgets/sorter/sorter.tsx

```diff
@@ -84,12 +77,14 @@ class Sorter extends React.Component<Props, State> {
         });
     };
 
-    getUserInput: () => {
-        options: any;
-    } = () => {
+    getUserInput: () => UserInput = () => {
         // eslint-disable-next-line react/no-string-refs
         // @ts-expect-error - TS2339 - Property 'getOptions' does not exist on type 'ReactInstance'.
-        return {options: this.refs.sortable.getOptions()};
+        const options = this.refs.sortable.getOptions();
```

Comment:
Had to pull this out so it is right beneath the `@ts-expect-error`. Wasn't exactly sure how to fix this hour, but it seemed like fixing it would take some larger refactoring, so left it alone.

Reply:
This is fine.  The reason for the `@ts-expect-error` is that `this.refs` is an old React ref system that TypeScript can't properly type. If we were to refactor, we'd make a field within this class named `sortableRef = React.createRef<Sortable>();` and then use that as the ref, which would then have proper types. 

I could pair with you this week to work through that refactoring if you'd like.